app/assets/javascripts/hyrax/collections.js

```diff
@@ -1,13 +1,12 @@
 Blacklight.onLoad(function () {
 
   // change the action based which collection is selected
-  $('input.updates-collection').on('click', function() {
- 
-      var form = $(this).closest("form"); 
-      var collection_id = $(".collection-selector:checked")[0].value;      
-      form[0].action = form[0].action.replace("collection_replace_id",collection_id);
+  // This expects the form to have a path that includes the string 'collection_replace_id'
+  $('[data-behavior="updates-collection"]').on('click', function() {
+      var string_to_replace = "collection_replace_id"
```

Comment:
If you feel strongly I can. I think this is silly because we have a eslint rule that says, we don't care about semicolins: https://github.com/samvera/hyrax/blob/master/.eslintrc#L183 It's just that hound isn't reading it right now.